Utilities/rXml.cpp

```diff
@@ -58,7 +71,7 @@ std::shared_ptr<rXmlNode> rXmlNode::GetChildren()
 
 std::shared_ptr<rXmlNode> rXmlNode::GetNext()
 {
-	wxXmlNode* result = reinterpret_cast<wxXmlNode*>(handle)->GetNext();
+	pugi::xml_node* result = &(reinterpret_cast<pugi::xml_node*>(handle)->next_sibling());
```

Comment:
Sorry about that. I changed that whole function, so that warning should not appear anymore. GetNext() looks like this now:
`
pugi::xml_node result = handle->next_sibling();
	if (result)
	{
		return std::make_shared<rXmlNode>(&result);
	}
	else
	{
		return std::shared_ptr<rXmlNode>(nullptr);
	}
`